src/cim/components/helpers/conditions.ts

```diff
@@ -0,0 +1,11 @@
+import { AgentK8sResource } from '../../types/k8s/agent';
+import { AgentClusterInstallK8sResource } from '../../types/k8s/agent-cluster-install';
+import { StatusCondition } from '../../types/k8s/common';
+
+export const getClusterValidatedCondition = (resource: AgentClusterInstallK8sResource) =>
+  resource.status?.conditions.find((c) => c.type === 'Validated') as StatusCondition<'Validated'>;
```

Comment:
nit: to be on the safe side: `conditions?.find()`

Reply:
done